### ISSUE TITLE
Filters Do Not Federate Across Instances

### ISSUE THREAD

**Issue by jimmybrancaccio** [pixelfed/pixelfed#2585]
Filters are not currently federating across instances. Here's a couple examples:

- Posted on my instance at [pixelfed.photos](https://pixelfed.photos/p/jimmyb/252485031426134016) does not show on [pixelfed.social](https://pixelfed.social/i/web/post/_/243908024434036736/252607168468094976).
- Posted on [pixelfed.social](https://pixelfed.social/p/jimmyb/252485855967580160) does not show on my instance at [pixelfed.photos](https://pixelfed.photos/i/web/post/_/244124496213184512/252485872958705664).

This has been brought up in IRC but I thought it be best to create a GitHub Issue so it's not forgotten about an is more "officially" known.

@dansup mentioned in IRC that they shouldn't be using CSS classes anymore, but that doesn't appear to be the case (or I misunderstood). For example, the HTML from the post on pixelfed.social:

```html
<img data-v-45b83517="" src="https://pixelfed.social/storage/m/_v2/243807968309678080/8fb78bfd7-ec88b5/JazfH3sO4eyT/7tLKQdSwfFory8obyx4XOf3EO603pA5g0KVfYjOm.jpg" loading="lazy" alt="Photo was not tagged with any alt text." onerror="this.onerror=null;this.src='/storage/no-preview.png'" class="filter-willow card-img-top">
```

**Comment by dansup** [comment]
@jimmybrancaccio This seems to be a regression, will investigate

**Comment by dansup** [comment]
Fixed in v0.10.10
